### PR TITLE
Added OpFlashes to SimpleFlash

### PR DESCRIPTION
**Pull request by Robert-Darby**
- Added OpFlashes for SBND and ICARUS
- - 2 flashes/slice for ICARUS (simpleflash, opflash)
- - Seperate training metrics for each flash
- - Both flashes recorded in CAFs (rec.slc.fmatch/fmatchop)
- UseCollection fcl parameter changed to PlaneList - select which wire planes hits are selected from (all by default)

### CODE REVIEW

**Comment by Robert-Darby** [comment]
Related PRs: 
icaruscode: https://github.com/SBNSoftware/icaruscode/pull/621
sbndcode: https://github.com/SBNSoftware/sbndcode/pull/381
sbncode: https://github.com/SBNSoftware/sbncode/pull/382
sbnanaobj: https://github.com/SBNSoftware/sbnanaobj/pull/107

File: fcl/configurations/flashmatch_simple_icarus.fcl [hunk 1]
```diff
@@ -1,19 +1,24 @@
-BEGIN_PROLOG #comment to run generate_simple_weighted_template.py
+# BEGIN_PROLOG #comment to run generate_simple_weighted_template.py
```
Comment (ikatza) [inline fcl/configurations/flashmatch_simple_icarus.fcl:1]: This line shouldn't be commented.

File: fcl/configurations/flashmatch_simple_icarus.fcl [hunk 2]
```diff
@@ -114,6 +119,17 @@ icarus_simple_flashmatch_W.PandoraProducer:    "pandoraGausCryoW"
 icarus_simple_flashmatch_W.TrackProducer:      "pandoraTrackGausCryoW"
 # icarus_simple_flashmatch_W.CaloProducer:     "pandoraGausCaloCryoW"
 icarus_simple_flashmatch_W.SpacePointProducer: "pandoraGausCryoW"
+icarus_simple_flashmatch_W.OpFlashProducer: ["opflashCryoE"]
+icarus_simple_flashmatch_W.OpFlashHitProducer: ["opflashCryoE"]
```
Comment (ikatza) [inline fcl/configurations/flashmatch_simple_icarus.fcl:123]: Shouldn't it be `opflashCryoW`?

**Review by ikatza — CHANGES_REQUESTED** [review]
Typos

**Comment by Robert-Darby** [comment]
New metrics file (ready for merge) at:

**Comment by Robert-Darby** [comment]
New ICARUS metrics file at: /icarus/data/users/rdarby/flashmatch/opflash/v09_77_00/train/FlashMatch

**Review by ikatza — APPROVED** [review]

**Comment by mmrosenberg** [comment]
trigger build SBNSoftware/sbncode#382 SBNSoftware/sbnanaobj#107

**Comment by mmrosenberg** [comment]
trigger build SBNSoftware/sbncode#382 SBNSoftware/sbnanaobj#107

**Comment by mmrosenberg** [comment]
trigger build SBNSoftware/sbncode#382 SBNSoftware/sbnanaobj#107

**Comment by mmrosenberg** [comment]
trigger build SBNSoftware/sbncode#382 SBNSoftware/sbnanaobj#107

**Pull request merged**